app/scripts/controllers/preferences.js

```diff
@@ -1,5 +1,6 @@
 const ObservableStore = require('obs-store')
 const normalizeAddress = require('eth-sig-util').normalize
+const isValidAddress = require('ethereumjs-util').isValidAddress
```

Comment:
Nit: You can save one level of indirection by doing `const { isValidAddress } = require('ethereumjs-util');`